pkg/trace/agent/agent.go

```diff
@@ -269,6 +272,25 @@ func (a *Agent) Process(p *api.Payload) {
 				sampler.SetPreSampleRate(root, rate)
 			}
 		}
+
+		if a.conf.ErrorTracking.Only {
+			var keep []*pb.Span
+			for _, span := range chunk.Spans {
+				if eligibleForErrorTracking(span) {
+					span.Meta[tagErrorTrackingMode] = "1"
```

Comment:
considered using `chunk.Tags` to set it once rather than on each span, but it doesn't seem to work like that.